lib/graphql/cache/resolver.rb

```diff
@@ -11,8 +11,8 @@ def initialize(type, field)
         @field = field
       end
 
-      def call(obj, args, ctx, &block)
-        resolve_proc = proc { block.call(obj, args, ctx) }
+      def call(obj, args, ctx, block)
+        resolve_proc = block #proc { block.call(obj, args, ctx) }
```

Comment:
is this change intentional?

Reply:
Yep, I left it because last two PRs added bunch of functionality and I think if I work on `{keep_fresh: true}`, I'll refactor new files a little bit.